### ISSUE TITLE
Parameterize infection date (possible for mutations / multiple strains?)

### ISSUE THREAD

**Issue by kkoenen** [DEUS-BV/covid19-dp3t-android#1]
Now, in `TriggerFragment.java`, the `onsetData` with which `DP3T.sendIWasExposed` is called, is hard coded to -14 days from `Calendar.DAY_OF_YEAR`. This could be implemented with more flexibility. I'm not a virologist, so I don't know if mutations could pop up with different incubation and spreading charateristics. intuitively, I'd implement a strain indicator (also in SK) to tie to a certain version of the virus. This strain indicator could be usefull in calculations, but also in more detailed reporting.

**Comment by DannyDeusAI** [comment]
Agree, we did mention this in some of the expert panels yesterday, also not sure we want a single number, we have a number of options here:
- keep to 14 as is, upside simplicity, downside you might inform more people than have been in range during the infectious period
- the GGD informs the user who is reporting ill during which period he may have infected others and we allow the user to set that in the app
- I also heard a new number of 7 yesterday during the conversation

TLDR: hardcoded for now, will update once it's clear
